### PR TITLE
port `tiddit-sv` to nf-test and add `--threads`

### LINKED ISSUE #5365: port `tiddit/sv` to nf-test

### PR DESCRIPTION
**Pull request by famosab**
## PR checklist

Closes #5365 and #5301 

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the module conventions in the [contribution docs](https://github.com/nf-core/modules/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, include test data in your PR.
- [x] Remove all TODO statements.
- [x] Emit the `versions.yml` file.
- [x] Follow the naming conventions.
- [x] Follow the parameters requirements.
- [ ] Follow the input/output options guidelines.
- [ ] Add a resource `label`
- [ ] Use BioConda and BioContainers if possible to fulfil software requirements.
- Ensure that the test works with either Docker / Singularity. Conda CI tests can be quite flaky:
  - For modules:
    - [ ] `nf-core modules test <MODULE> --profile docker`
    - [ ] `nf-core modules test <MODULE> --profile singularity`
    - [ ] `nf-core modules test <MODULE> --profile conda`
  - For subworkflows:
    - [ ] `nf-core subworkflows test <SUBWORKFLOW> --profile docker`
    - [ ] `nf-core subworkflows test <SUBWORKFLOW> --profile singularity`
    - [ ] `nf-core subworkflows test <SUBWORKFLOW> --profile conda`

### CODE REVIEW

**Comment by famosab** [comment]
@nf-core-bot fix linting

File: modules/nf-core/tiddit/sv/meta.yml [hunk 1]
```diff
@@ -21,14 +21,24 @@ input:
...
-  - index:
+  - input_index:
       type: file
       description: BAM/CRAM index file
       pattern: "*.{bai,crai}"
+  - meta2:
+      type: map
+      description: |
+        Groovy Map containing sample information
+        e.g. [ id:'test', single_end:false ]
```
Comment (mashehu) [inline modules/nf-core/tiddit/sv/meta.yml:32]:
```suggestion
        e.g. `[ id:'test', single_end:false ]`
```

File: modules/nf-core/tiddit/sv/meta.yml [hunk 2]
```diff
@@ -21,14 +21,24 @@ input:
...
   - fasta:
       type: file
       description: Input FASTA file
       pattern: "*.{fasta,fa}"
+  - meta3:
+      type: map
+      description: |
+        Groovy Map containing sample information from bwa index
+        e.g. [ id:'test', single_end:false ]
```
Comment (mashehu) [inline modules/nf-core/tiddit/sv/meta.yml:41]:
```suggestion
        e.g. `[ id:'test', single_end:false ]`
```
please double check if this meta map example make sense in their context.

**Review by mapo9 — APPROVED** [review]
lgtm!

**Review by mashehu — APPROVED** [review]

File: modules/nf-core/tiddit/sv/meta.yml (hunk 1 above)
Comment (famosab) [inline modules/nf-core/tiddit/sv/meta.yml:32]:
```suggestion
        e.g. `[ id:'test_fasta']`
```

File: modules/nf-core/tiddit/sv/meta.yml (hunk 2 above)
Comment (famosab) [inline modules/nf-core/tiddit/sv/meta.yml:41]:
```suggestion
        e.g. `[ id:'test_bwa-index' ]`
```

File: modules/nf-core/tiddit/sv/meta.yml (hunk 1 above)
Comment (famosab) [reply modules/nf-core/tiddit/sv/meta.yml:32]: @mashehu Does that make more sense?

Comment (mashehu) [reply modules/nf-core/tiddit/sv/meta.yml:32]: jep

**Pull request merged**